addons/mrp_repair/models/mrp_repair.py

```diff
@@ -225,6 +225,40 @@ def action_repair_cancel(self):
         return self.write({'state': 'cancel'})
 
     @api.multi
+    def action_send_mail(self):
```

Comment:
same as above why we need to open compose wizard and email template to just send quotation ?